requirements/lint.txt

```diff
@@ -152,8 +152,6 @@ faker==17.4.0
     #   -r requirements/functests.txt
     #   -r requirements/tests.txt
     #   factory-boy
-filelock==3.13.1
-    # via -r requirements/tests.txt
```

Comment:
No more `filelock` dependency since the file-based locking mechanism is no longer needed when initializing the DB.